src/Microsoft.DotNet.ProjectModel/Project.cs

```diff
@@ -120,6 +120,15 @@ public TargetFrameworkInformation GetTargetFramework(NuGetFramework targetFramew
             return targetFrameworkInfo ?? _defaultTargetFrameworkConfiguration;
         }
 
+        public bool HasRuntimeOutput(string configuration)
+        {
+
+            var compilationOptions = GetCompilerOptions(targetFramework: null, configurationName: configuration);
+
+            // TODO: Make this opt in via another mechanism
+            return compilationOptions.EmitEntryPoint.GetValueOrDefault() ||
+                   !string.IsNullOrEmpty(TestRunner);
```

Comment:
Add the IsTestRunner property back. You can add it in there. Is it much better than this negation to reach a positive.